### DISCUSSION TITLE
Troubleshooting Supabase Realtime: Events Not Triggering and Payload Not Displaying in Console

### DISCUSSION THREAD

**Discussion by panshoku** [supabase/supabase#18907] [Questions]
I have implemented a subscription as shown below, but when I modify data, the event does not trigger, and the payload is not displayed in the console.

In the Supabase dashboard, I have made the following settings:
1. Database > Tables > samples (target table name) > Realtime Enabled
2. Database Replications > On (Insert, Update, Delete, Truncate)
3. Database Replications > Source Column > On samples Table

Do you have any idea what might be causing this issue?

```javascript
// boot/supabase.js
import { createClient } from '@supabase/supabase-js';

let supabaseConfig = {
    url: 'https://xxxxxxxx.supabase.co',
    key: 'xxxxxxxx',
};

let supabase = null;

export const createSpabase = (supabase_token) => {
    const newClient = createClient(supabaseConfig.url, supabaseConfig.key, {
    global: {
        headers: {
            Authorization: `Bearer ${supabase_token}`,
        },
    },
    });

    supabase = newClient;
};

export { supabase };


// src/stores/samples.js
let channel = null;

export const useSamplesStore = defineStore({
    id: 'samples',
    actions: {
        async start () {
            this.stop();
            channel = supabase
              .channel('samples')
              .on(
                'postgres_changes',
                {
                  event: '*',
                  schema: 'public',
                  table: 'samples',
                },
                async (payload) => {
                 // Do not show !
                 console.log('payload: ', payload);
                },
              )
              .subscribe((status, err) => {
                if (status === "SUBSCRIBED") {
                  // Is displayed
                  console.log('SUBSCRIBED');
                }
      
                if (status === "CHANNEL_ERROR") console.log('CHANNEL_ERROR');
      
                if (status === "TIMED_OUT") console.log('CHANNEL_ERROR');
              });
          },
          stop() {
            if (channel) {
              supabase.removeChannel(channel);
              channel = null;
            }
          },
    }
})

// Sample.vue
<script setup>
import { useSamplesStore } from '@/stores/samples';
import { onMounted, onUnmounted } from 'vue';

onMounted(() => {
    useSamplesStore().start();
}

onUnmounted(() => {
    useSamplesStore().stop();
}
</script>
```

**Comment by ghost** [comment]
We are facing a problem in our project. It appears that there are some issues with Supabase Realtime Server.

**Comment by GaryAustin1** [comment]
Did this resolve?
Also make sure your user meets select RLS policy on the table.
This is the realtime status page and has an inspector you can use to make sure it is not your code causing the issue:  https://realtime.supabase.co/

**Comment by PollieDev** [comment]
Im having a similar problem at the moment. Strangely, from my end it works perfectly fine on one table but not the others.
Altough the setup of both tables are pretty much identical.

On code-level both tables are being called the exact same way, they use the same piece of code.
On dashboard-level both tables have RLS enabled with the same rules, I am perfectly able to fetch data from both tables but I'm not receiving realtime updates on one of them.
`supabase_realtime` is also enabled and both tables are selected to use it.

Am I missing anything else that I'd need to check wether both tables have everything required? I'm very confused why it works perfectly on one but not the other. Either I did miss an option somewhere or there's something strange going on with the realtime updates for me.
